### ISSUE TITLE
Build Tools Doc Update 

### ISSUE THREAD

**Issue by tinefin** [pantheon-systems/documentation#3214]
Re: [Build Tools](https://pantheon.io/docs/guides/build-tools/)

Priority (Low‚ Medium‚ High):

## Issue Description:

Had a customer reporting this error:

```
$ composer create-project -n -d $HOME/.terminus/plugins pantheon-systems/terminus-composer-plugin:~1


[RuntimeException] 
Invalid working directory specified, /home/www-data/.terminus/plugins does not exist. 
```

Internal issue id: 14060483235

## Suggested Resolution 

If this is a general issue, the line should probably be changed to something like:
`$ mkdir $HOME/.terminus/plugins; composer create-project -n -d $HOME/.terminus/plugins pantheon-systems/terminus-composer-plugin:~1`

(please test it - I haven't)

**Comment by sarahg** [comment]
Looks like this has already been addressed here: https://pantheon.io/docs/guides/build-tools/ (Step 4: Create the $HOME/.terminus/plugins directory if it does not already exist)

**Comment by alexfornuto** [comment]
@sarahg is correct, this was resolved in #3294
